escalation/application.properties

```diff
@@ -26,8 +26,12 @@ quarkus.kubernetes-client.trust-certs=true
 quarkus.rest-client.notifications.url=${BACKSTAGE_NOTIFICATIONS_URL:http://backstage-backstage.rhdh-operator}
 quarkus.openapi-generator.notifications.auth.BearerToken.bearer-token=${NOTIFICATIONS_BEARER_TOKEN}
 
-# This is to enable debugging of HTTP request 
-quarkus.log.category.\"org.apache.http\".level=INFO
+# This property is used to select the log level, which controls the amount
+# of information logged on HTTP requests based on the severity of the events. 
+# Possible values: OFF, FATAL, ERROR, WARN, INFO, DEBUG, TRACE, ALL. 
```

Comment:
Pls note that DEBUG is the most verbose possible value. TRACE isn't supported in prod mode, therefore we should remove that option from the list.

Reply:
here and in the other files :-)